core/api_requests.py

```diff
@@ -50,36 +51,15 @@ def get_history(id):
     response = requests.post(URL, json={'query': query_history(id)})
 
     return response.json()
-
 def get_rockets():
     """
     request rockets information
     """
     response = requests.post(URL, json={'query': query_rockets()})
     return response.json()
-
 def get_rocket(id):
     """
     request single rocket information
     """
     response = requests.post(URL, json={'query': query_rocket(id)})
-    return response.json()
-
-
-def get_missions():
```

Comment:
## pt

Por que remover a consulta de missoes?

## en
Why remove the missions query?